cs244b_project/src/lib.rs

```diff
@@ -632,7 +648,7 @@ impl StreamletInstance {
         None 
     }
 
-    /* Determines if the block associated with a message is notarized.
+    /* Determines if the leader was first to sign this message. Used for justifying a new vote. 
```

Comment:
I actually think we only need to call this when we receive a proposal, not when we receive a vote!